components/AccountWidget/AccountWidget.jsx

```diff
@@ -7,18 +7,18 @@ import { Link } from 'react-router';
 
 // Maybe we can factor out this ternary into a function?
 const AccountWidget = ({ loading, error, player }) => (
-  <div>
+  <menu>
```

Comment:
Yeah. It's part of HTML5.  There's a bunch of them. section, article, header, footer (and more).  They're all just block-level elements (divs) but hold semantic value for search engines (not that the SPA will work with search engines, unless we attempt to implement server-side rendering at a later date).